IFC4x3/Properties/c/Ca2+_10wmfjd5PB5xuqHQBOdisp/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Ca2+_10wmfjd5PB5xuqHQBOdisp" Name="Ca2+" UniqueId="40eb0a6d-9c56-4b17-be34-45a2d89ecdb3" PrimaryDataType="IfcReal" />
```

Comment:
How about we rename this to "WaterHardness" or "CalciumContent"?

Reply:
According to discussions with domain experts, we keep the properties from Pset_ChemicalLabTestLiquid in Pset_MaterialWater. Not certain that WaterHardness and CalciumContent refers to the same.